common/cgroup/cgroup.go

```diff
@@ -253,7 +274,9 @@ func CreateCgroups(root string, subcgroup string, serviceNames []string) error {
 		syscall.MS_NOSUID |
 		syscall.MS_NOEXEC |
 		syscall.MS_NODEV
-	if err := syscall.Mount("sysfs", sys, "sysfs", flags, ""); err != nil {
+	// ignore EBUSY errors: if we're creating the cgroups on the host it
+	// could be mounted already
+	if err := syscall.Mount("sysfs", sys, "sysfs", flags, ""); err != nil && err != syscall.EBUSY {
```

Comment:
If the `/sys/fs/cgroup` tree is partially mounted (e.g. the tmpfs is mounted read-only by the distribution but not the controllers), it will not work because the tmpfs is read-only. So isn't it better to remove the EBUSY checks and instead have a single check at the beginning of the function `CreateCgroups` to see if the work is already done?